FSI/Displacement.C

```diff
@@ -2,22 +2,35 @@
 
 using namespace Foam;
 
-preciceAdapter::FSI::Displacement::Displacement
-(
-    const Foam::fvMesh& mesh,
-    const std::string namePointDisplacement
-)
-:
-pointDisplacement_(
-    const_cast<pointVectorField*>
-    (
-        &mesh.lookupObject<pointVectorField>(namePointDisplacement)
-    )
-)
+preciceAdapter::FSI::Displacement::Displacement(
+    const Foam::fvMesh &mesh,
+    const std::string   namePointDisplacement)
+    : pointDisplacement_(
+          const_cast<pointVectorField *>(
+              &mesh.lookupObject<pointVectorField>(namePointDisplacement))),
+      cellDisplacement_(
+          const_cast<volVectorField *>(
+              &mesh.lookupObject<volVectorField>("cellDisplacement"))),
```

Comment:
For consistency, we should also get the name of `cellDisplacement` from the configuration file. Even though I guess we could eventually drop this feature, I am not sure if anyone is using it.

Reply:
Which configuration file? The preciceDict? Or should I just store the name in the `FSI.H`? Note that we actually read an already present `volVectorField` called `cellDisplacement`.

Reply:
The `preciceDict`. Instead of `"cellDisplacement"` we should have something like `nameCellDisplacement`, as in three lines above.

Reply:
Done